bin/index.js

```diff
@@ -104,50 +101,25 @@ const askQuestions2 = () => {
       type: 'list',
       name: '_actionType',
       message: 'What action is the certificate asserting?',
-      choices: ["REGISTER", "UPDATE"],
+      choices: ["REGISTER", "DE_REGISTER"],
       default: "REGISTER"
     },
     {
-      name: "_releaseNumber",
+      name: "_releaseComment",
       type: "input",
-      message: "What's the release number?",
-      default: '0.0.1'
+      message: "Enter a release comment:",
```

Comment:
Don't know how to mark the optional field. Maybe there is a better way
```suggestion
      message: "Comment your change (optional)",
```

Reply:
Fixed on d41e940